tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/LiferayGradleProject.java

```diff
@@ -346,4 +347,21 @@ public boolean isFragmentBundle()
         return false;
     }
 
+    @Override
+    public boolean isThemeBundle()
+    {
+        IProject project = getProject();
+        final CustomModel model = GradleCore.getToolingModel( GradleCore.getDefault(), CustomModel.class, project );
+        Set<String> pluginClassNames = model.getPluginClassNames();
```

Comment:
This could be really really slow, we will need a faster way to find this information out.  This class can only be done at once when project is first imported or reconfigured.  It can't be done easiy in a wizard since it could take a long time.